extendr-api/src/wrapper/environment.rs

```diff
@@ -127,6 +129,7 @@ impl Environment {
         }
     }
 
+    #[cfg(feature = "non-api")]
     /// Get the names in an environment.
     /// ```
     /// use extendr_api::prelude::*;
```

Comment:
Doc breaking change for `set_local()` method. Goes in changelog

Reply:
This is missing from changelog too